lib/scala/filewatcher/src/main/java/org/enso/filewatcher/DefaultWatcherFactory.java

```diff
@@ -0,0 +1,29 @@
+package org.enso.filewatcher;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.WatchService;
+import java.util.function.Consumer;
+import org.enso.filewatcher.Watcher.WatcherError;
+import org.enso.filewatcher.Watcher.WatcherEvent;
+
+public final class DefaultWatcherFactory implements WatcherFactory {
```

Comment:
```suggestion
final class DefaultWatcherFactory implements WatcherFactory {
```
add static method into `WatcherFactory.createDefault()` to return this instance.

Reply:
Done in 5679066de9c3469cea53b3ec497068ba764ae4d5